src/xray/jest-xray-environment.ts

```diff
@@ -0,0 +1,37 @@
+import NodeEnvironment from "jest-environment-node";
+
+class JestXRayEnvironment extends NodeEnvironment {
+
+  override async setup() {
+    await super.setup();
+  }
+
+  async handleTestEvent(event: any) {
```

Comment:
You can define an interface you depend that you use here instead of `any` so that you don't drop out the type system ;) (General guideline: omit `any` wherever possible, a better alternative if something is too hard to type, e.g., as an interface, or really unknown is `unknown`)